bertopic/_bertopic.py

```diff
@@ -73,12 +72,23 @@
 )
 import bertopic._save_utils as save_utils
 
-# Visualization
-import plotly.graph_objects as go
 
 logger = MyLogger()
 logger.configure("WARNING")
 
+try:
+    from bertopic import plotting
+    import plotly.graph_objects as go
+
+except ModuleNotFoundError as e:
+    if "No module named 'plotly'" in str(e):
+        logger.warning("Plotly is not installed. Please install it to use the plotting functions.")
```

Comment:
Perhaps also add a way to install it? A one-liner would suffice, something like "For example with `pip install plotly` or `uv pip install plotly`". That way, users would not have to search how to install plotly but can do it immediately.